specification/gp-connect-appointments-management-fhir.yaml

```diff
@@ -6,118 +6,98 @@ info:
   version: 'Computed and injected at build time by `scripts/set_version.py`'
   description: |
     ## Overview
-    Use this API to enable patients to book and manage patient appointments held in any of the principal GP practice systems.
 
-    You can:
+    Use this API to book and manage patient appointments at their GP practice.
 
-    * retrieve a patient’s appointments
-    * search for free slots
-    * read an appointment
-    * book an appointment
-    * cancel an appointment
+    You can:
+     - retrieve a patient’s appointments
+     - search for free slots
+     - read an appointment
+     - book an appointment
+     - cancel an appointment
 
-    For example:
+    You cannot:
+     - amend a patient's appointments
+     - re-schedule a patient's appointments (This must be done by cancelling and re-booking)
 
-    * A patient can book, view, or cancel their own appointments.
+    The end user must be a patient who is:
+     - registered with the GP practice
+     - registered with NHS login to P9
 
-    Note: You need to use this API in conjunction with the APIM API which will handle
+    ## For Example:
 
-    * Getting the patient details
-    * Determining the patient's GP
-    * Security protocols
+    A patient can add, remove or retrive their appointments
 
-    Add a fully fledged description here with markdown syntax.
-    For help completing the content of your API spec, see our guidance on the minimum standard (bronze) details
-    to include [here](https://nhsd-confluence.digital.nhs.uk/display/APM/Writing+a+bronze+API+cover+page). You should also base your content on our exemplar API - [PDS FHIR](https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir).
     ## Who can use this API
 
-    This API can be used by developers of clinical systems that support clinicians delivering direct care. It is not a patient-facing API.
-
-    Before you start development, read the [Appointment Management](https://simplifier.net/guide/pfs-appointments?version=current) section of the GP Connect v1.2.7 specification, which includes Foundations, and the prerequisites listed below.
+    This API can be used by developers of clinical systems that support clinicians delivering direct care. It is a patient-facing API.
```

Comment:
Have a look at https://github.com/NHSDigital/gp-connect-user-permissions/blob/master/specification/gp-connect-user-permissions.yaml#L48 for what user-permission has in this section.